libs/model/src/services/openai/generateTokenIdea.ts

```diff
@@ -58,17 +60,72 @@ const chatWithOpenAI = async (prompt = '', openai: OpenAI) => {
   });
   convoHistory.push(response.choices[0].message); // assistant msg
 
-  return (response.choices[0].message.content || 'NO_RESPONSE').replace(
+  const result = (response.choices[0].message.content || 'NO_RESPONSE').replace(
     /^"|"$/g, // sometimes openAI adds `"` at the start/end of the response + tweaking the prompt also doesn't help
     '',
   );
+  console.log('Received response from OpenAI:', result);
+  return result;
+};
+
+const generateImageWithRunware = async (prompt: string) => {
```

Comment:
This is duplicated from `generateImage.ts`. Functions should be moved to a single `generateImage` util function in `models/src/utils` so that code remains dry and we can easily mock in testing (`config` check should also happen in the util function so that generating an image is super simple from the perspective of the callee).